code/game/gamemodes/blob/blobs/blob_mobs.dm

```diff
@@ -271,3 +275,7 @@
 		factory.naut = null //remove this naut from its factory
 		factory.maxhealth = initial(factory.maxhealth)
 	flick("blobbernaut_death", src)
+
+/mob/living/simple_animal/hostile/blob/blobbernaut/independent
```

Comment:
It has nothing to do with shuttles, but I discovered that a bolt of changed blobbernaut would start dying while making and testing the wabbajack shuttle.